### ISSUE TITLE
Users can duplicate post titles

### ISSUE THREAD

**Issue by karlgroves** [WordPress/gutenberg#15282]
Users can duplicate post titles

-   **Severity:** Medium
-   **Affected Populations:**
    -   Blind
    -   Low-Vision
    -   Cognitively Impaired
-   **Platform(s):**
    -   All / Universal
-   **Components affected:**
    -   Output Markup

#### Issue description

It's possible for authors to create multiple posts with the same page
title.

Assistive technology users may rely on the title to understand the
primary purpose of each page. When titles are not unique, it may be more
difficult and laborious for such users to determine what the page is
about.

#### Remediation Guidance

Having something warn authors that another post has already been
published (not drafted) with the same title would help authors ensure
post titles are unique.

This need not literally prevent users from creating multiple posts with
the same title, merely warn them why this is not a good idea.

#### Relevant standards

-   [2.4.2 Page Titled
    (Level A)](https://www.w3.org/TR/WCAG20/#navigation-mechanisms-title)

**Note:** This issue may be a duplicate with other existing accessibility-related bugs in this project. This issue comes from the Gutenberg accessibility audit, performed by [Tenon](https://www.tenon.io) and funded by [WP Campus](https://wpcampus.org/). This issue is GUT-112 in Tenon's report

**Comment by afercia** [comment]
An accessible warning to inform users there's already a post with the same title seems a very interesting idea. Also from an SEO perspective.

**Comment by kjellr** [comment]
We could theoretically open up a notification like this: 

<img width="2420" alt="Frame" src="https://user-images.githubusercontent.com/1202812/57400396-60d8b900-71a1-11e9-91e5-a479cd799899.png">

I'm not sure if it makes sense to explain why duplicate titles aren't great — bringing terms like "Accessibility" and "SEO" into the mix may further confuse things.

**Comment by mapk** [comment]
As mentioned in today's design triage in slack, this might be a great opportunity to use the new snackbar notifications being explored by @youknowriad right now in #15594. 

>Since it’s in the category of SEO, perhaps snackbar is the right amount of “intrusive”? We should probably _guide_ and not _bug_ people

Once that PR is merged, let's bring that pattern here. Does this work for everyone?

**Comment by afercia** [comment]
First time I hear about the explorations on "snackbars". While I see some accessibility considerations were reported on the related issue #13905, I'd like to ask why the issue and the PR #15594 haven't been labelled with the "Needs Accessibility Feedback" label. This appears to be a pretty important change in the UI and in the way visual/audible feedback is provided to users.

I'm a bit surprised that, while there's ongoing discussion about ways to improve collaboration between teams and improve the overall development process, the accessibility team hasn't been involved in the evaluation of this proposal _before_ development actually started. I'd greatly appreciate some feedback. /Cc @youknowriad

**Comment by mapk** [comment]
@afercia let's assume positive intent. If you'll notice @youknowriad's PR doesn't have ANY labels at this point. There are no "Needs Design Feedback" labels and yet Joen somehow came across it and began to interact with it in a postive way by leaving comments. Riad is doing excellent work exploring new ideas. Let's help him out! This is what collaboration is. 😉 

I'd like to note that Riad's PR is purely experimental right now. So I'll take complete blame on suggesting its possible merge which may have caused some alarm.

**Comment by afercia** [comment]
For the greater good of this project, I'll refrain from further comments. I'd just like to note this is more about process to improve and trust to be rebuilt.

**Comment by youknowriad** [comment]
>  haven't been labelled with the "Needs Accessibility Feedback"

That's just because it's a draft PR :) not labeled and not ready yet :)

**Comment by nerrad** [comment]
What about using the pre-publish checks sidebar for this for posts that aren't published yet?  If this is disabled, then use notices.  Even if notices are still used, the pre-publish check might still be worthwhile for having a summary of accessibility related issues as a "are you sure" kind of thing.

**Comment by skorasaurus** [comment]
duplicate of https://github.com/WordPress/gutenberg/issues/11241
